src/models/applicationState.ts

```diff
@@ -1,3 +1,6 @@
+import { TagsDescriptor } from "vott-ct/lib/js/CanvasTools/Core/TagsDescriptor";
```

Comment:
Still working to fix this. For now, since the library exports are wrapped in a single canvastools module, vscode defaults to this. Otherwise you can import canvastools module and use relative calls for types